.github/workflows/upload.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         env:
           CI: false
       - name: Compress
-        run: cd build && tar -czf ../${{env.PACKAGE_NAME}} ./
+        run: cd build && tar -czf ../${{env.PACKAGE_NAME}} ./ && ls -hs ../${{env.PACKAGE_NAME}} 
```

Comment:
@sparkiegeek  Might be useful to temporarily update lines 3-4 in this file

from
```
on:
  pull_request_target:
    types: [closed]
```

to
```
  pull_request:
    branches:
      - main
```

to get results without having to merge this.

Reply:
Oops, forgot that you need to change this line too:

https://github.com/canonical/maas-ui/pull/4628/files#diff-8cf0b5fae548aab9bd49f5020da9c0e35d281984b82b8a5358ffb1c1ae5bec13L7